testing/test_pluginmanager.py

```diff
@@ -350,3 +351,17 @@ def m(self, __multicall__, x):
 def test_add_hookspecs_nohooks(pm):
     with pytest.raises(ValueError):
         pm.add_hookspecs(10)
+
+
+def test_reject_prefixed_module(pm):
+    """Verify that a module type attribute that contains the project
+    prefix in its name (in this case `'example_*'` isn't collected
+    when registering a module which imports it.
+    """
+    pm._implprefix = 'example'
+    conftest = types.ModuleType("conftest")
```

Comment:
For completeness, I would add an `example_` function to one of the modules and ensure that it *is* collected.

Reply:
Done. Heh, and found a bug from the `_Multicall` refactor.. shocker we've got no tests for the undecorated hook collection stuff.